### ISSUE TITLE
Large-scale metagenomic analysis of oral microbiomes reveals markers for autism spectrum disorders

### ISSUE THREAD

**Issue by SvetlanaUP** [waldronlab/BugSigDBcuration#525]
Large-scale metagenomic analysis of oral microbiomes reveals markers for autism spectrum disorders – Paolo Manghi – Nature Communications
https://www.nature.com/articles/s41467-024-53934-7

**Comment by KateRasheed** [comment]
Good evening @SvetlanaUP . Please can I curate this article 😊.

**Comment by KateRasheed** [comment]
Good evening @SvetlanaUP . I tried retrieving the PMID for this article but it is giving zero result on PubMed.

**Comment by SvetlanaUP** [comment]
Good morning @KateRasheed 
Above the title we can see
Article
_Open access Published: **11 November 2024**_
that this paper is published very recently, on Monday, two days ago.

Sometimes it needs time to get PMID assigned. 

Continue with your curation and check in a few days if there is PMID. Once out we will add it to the curation.

**Comment by KateRasheed** [comment]
I thought as much. Thank you so much @SvetlanaUP for the clarification.

**Comment by KateRasheed** [comment]
Good day @SvetlanaUP . This study is ready for review:

https://bugsigdb.org/Study_1218

Thank you.

**Comment by AleruDivine** [comment]
Well done on your curation @KateRasheed, I remember this study and you did a really good job! It's a lot of signatures and I know you put in a lot of work in curating them. Well done!

That said, I have noted a few adjustments and clarifications that I think will help improve the study:

**Missing Signatures:**
1. **Neisseria subflava** is missing from Signature 2 of Experiment 1. _Neisseria flava_ was curated instead. Please verify and correct.
2. _Fusobacterium periodonticum_ is missing from Signature 2 of Experiment 2.


**Adjustments Needed:**
1. In Signature 1 of Experiment 4 (Fig. 3c), _Prevotella sp. oral taxon 306_ has a q-value > 0.2 and should be removed.
2. For Signature 2 in Experiment 4, the following taxa also have q-values > 0.2 and need to be removed too:
**_Pauljensenia hongkongensis
Rothia dentocariosa
Streptococcus sp. oral taxon 056
Schaalia georgiae
Corynebacterium durum_**

See excerpt "Considering the top-15 species either positively or negatively associated with ASD in the original cohort, 18 were still significantly associated (q < 0.2) in the differential abundance model run in the subset, but none was associated with ARFID score and Picky Factor, while **24 remained significantly associated with IQ** (Fig. [3c](https://www.nature.com/articles/s41467-024-53934-7#Fig3), Supplementary Fig. [7b](https://www.nature.com/articles/s41467-024-53934-7#MOESM1))" 

It seems the reason you curated the signatures above is because they were in supplemental data 13. 

Please note this, For experiment 4 (unadjusted for picky factor), the supplemental data to curate is Supplemental Data 18 and for experiment 5 (adjusted for picky factor), the supplemental data to curate is Supplemental Data 17. They are in agreement with figure 3C.

Please see excerpt "We ran three separate case-only models, assessing the correlation between oral microbiome composition and ARFID Score, Picky Factor and imputed IQ. When comparing these models with those of the differential abundance model on the subset, the ARFID Score betas showed a low correlation with ASD (ρ = 0.11, p = 1.1 × 10−4), and the Picky Factor score showed a slightly higher correlation (ρ = 0.22, p = 10−16), but neither ARFID nor the Picky Factor scores showed significant associations at q < 0.2 (Suppl. Data [15](https://www.nature.com/articles/s41467-024-53934-7#MOESM3), [16](https://www.nature.com/articles/s41467-024-53934-7#MOESM3)). Interestingly, IQ showed the highest number of statistical associations, either including or not including Picky Factor in the same model (n = 94 and 84, respectively, Suppl. Data [17](https://www.nature.com/articles/s41467-024-53934-7#MOESM3), [18](https://www.nature.com/articles/s41467-024-53934-7#MOESM3)), as well as the highest correlation with the differential abundance model run in the subset (ρ = −0.33, p = 1.8 × 10−34, Fig. [3b](https://www.nature.com/articles/s41467-024-53934-7#Fig3)), suggesting a major role in the observed ASD-related microbiome differences with respect to dietary habits."

Let me know what you think, thank you so much!

**Comment by KateRasheed** [comment]
Thanks for the review. I'll look through and resolve as soon as I can.

**Comment by KateRasheed** [comment]
Good morning @AleruDivine . I curated Supplementary Data 13 because it has significant taxa. See this: "After adjusting the association with IQ by the estimated microbial load, a total of 131 significant associations with IQ (q < 0.005) were found. While 44/131 significant associations were linked to both, only 14 species were positively associated with microbial load and negatively with IQ, suggesting IQ and oral hygiene as at least partially independent drivers of oral microbiome composition. Notably, the betas for the IQ species association were highly correlated with the ASD betas (ρ = −0.51, p = 2.7 × 10−88, Suppl. Data [13](https://www.nature.com/articles/s41467-024-53934-7#MOESM3)"

I think this should be curated as a separate experiment. I'll change the significant threshold for experiment 4 and adjust the taxa as well.

I'll curate a separate experiment for Supplementary Data 17 since it has a different q-value threshold.

For the adjustment by picky factor, Supplementary Data 18 not 17 is to be curated and I did that. Please check again.

Let me know your thoughts on this. Thank you so much.

**Comment by AleruDivine** [comment]
@KateRasheed Yes I agree with curating Supplemental data 13 and 17 as separate experiments. It makes sense. For supplemental data 17, you should combine with 3c not data 13.
Also correct source for signature 1 in experiment 5 (add supplementary data 18), you only added that in the second signature.
Thanks!

**Comment by KateRasheed** [comment]
Alright. Thank you so much.

**Comment by KateRasheed** [comment]
Good morning @AleruDivine . All corrections have been made on this study. Thank you so much.

**Comment by SvetlanaUP** [comment]
https://bugsigdb.org/Study_1218 reviewed.
